web/src/main/webapp/WEB-INF/data/data/formatter/xslt/render-layout.xsl

```diff
@@ -269,15 +269,15 @@
   -->
   <xsl:template mode="render-view"
                 match="section[@xpath]">
-    <div id="gn-view-{generate-id()}">
-      <xsl:apply-templates mode="render-view" select="@xpath"/>&#160;
```

Comment:
Same as icon here I think. If the apply-templates return null content, the HTML produced will be invalid and we have layout issues. Test the HTML mode only of formatter API and not only the formatter embedded in angular apps.

Reply:
I tested this, and when the `&#160;` is removed the content of the tab is empty and the whole tab-pane is not added. When you leave the `&#160;`, the tab is not empty and added. If you display this tab only the title is shown.

Reply:
I'm going to do some extra checks to see if the HTML is well formed.

Reply:
I double checked the HTML and everything looks good and valid. When the `&#160;` is removed the tab is empty and not included.